docs/source/conf.py

```diff
@@ -182,13 +200,30 @@
 
 
 # -- Extension configuration -------------------------------------------------
-autodoc_mock_imports=["tensorflow", "sklearn", "skimage", "nbformat", "cv2", "keras_retinanet", "keras_maskrcnn"]
+autodoc_mock_imports = ["tensorflow", "sklearn", "skimage",
+                        "nbformat", "cv2", "keras_retinanet", "keras_maskrcnn"]
 
 # -- Options for intersphinx extension ---------------------------------------
 
 intersphinx_mapping = {
-    'deepcell': ('https://deepcell.readthedocs.io/en/documentation/', None),
+    'deepcell': ('https://deepcell.readthedocs.io/en/{}/'.format(rtd_version), None),
+    'redis_consumer': ('https://deepcell-kiosk.readthedocs.io/projects/kiosk-redis-consumer/en/{}/'.format('mrgn-docs-autodoc'), None),
```

Comment:
again, seems like we shouldn't have these custom branches in the PR.

Reply:
Changed to reference `rtd_version`